### PR TITLE
Update perf-data.adoc

### PR DESCRIPTION
**Pull request by mmahacek**
### All Contributors

* [X] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### External References

* JIRA (Issue Tracker): No JIRA

### CODE REVIEW

**Comment by mmahacek** [comment]
This includes some updates to the perf-data file as well as cross-linking the page in the collection section, since it is related to performance data collection and persisting metrics.

File: docs/modules/operation/pages/events/perf-data.adoc [hunk 1]
```diff
@@ -162,24 +163,35 @@ report.uei.opennms.org_traps_test_varbind.command=--title="uei.opennms.org_traps
...
-If your time series data type is not nodeSnmp/interfaceSnmp, you need to create a resource-type file in etc/resource-types.d.
-Make sure you match the name with evenconf's resourceType.
-There is also ${instance} variable available to provide a name to help indicate what an event binds to.
+=== Create resource-types (optional)
 
-Example:
-```
+If your time series data type is not `nodeSnmp` or `interfaceSnmp`, you need to create a resource-type file in `$\{OPENNMS_HOME}/etc/resource-types.d`.
+Make sure you match the name with the event definition's `resourceType`.
+There is also `$\{instance}` variable available that can be used to enrich the resource type's label.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/events/perf-data.adoc:175]:
```suggestion
There is also `$\{instance}` variable available that you can use to enrich the resource type's label.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
minor change. Thanks!

**Review by Bonrob2 — APPROVED** [review]
SHIP IT!

**Comment by mmahacek** [comment]
@freddy61025 Can you verify these changes are still representing accurate information?  Thanks.

**Pull request merged**